sidebar.md

```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 ##Filter Posts
 [MOD POST](https://www.reddit.com/r/AsianBeauty/search?q=+flair:mod-post&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [REVIEW](https://www.reddit.com/r/AsianBeauty/search?q=+flair:review&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[PSA](https://www.reddit.com/r/AsianBeauty/search?q=+flair:psa&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [QUESTION](https://www.reddit.com/r/AsianBeauty/search?q=+flair:question&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[SWATCHES & FOTD](https://www.reddit.com/r/AsianBeauty/search?q=+flair:swatches-fotd&restrict_sr=on&sort=new&t=all&feature=legacy_search)

```

Comment:
I think we should revisit FOTD only but with a few more rules around how to post ie general routine and what the photo is a of/ a blurb